### PR TITLE
Cop 12096/checkbox spacing

### PR DESCRIPTION
**Pull request by bowerscodes-COP**
- Restructured FormGroup to include fieldset and legend tags to meet accessibility requirements, and removed the resulting duplicate margin from the legend.
- Updated tests for FormGroup, DateInput and TimeInput, which were affected by the change to the child nodes of the rendered component

[Jira Ticket](https://support.cop.homeoffice.gov.uk/browse/COP-12096)

### CODE REVIEW

**Review by aliktb — COMMENTED** [review]
Could you just provide a brief description of the issue/link the ticket so we have a record in github. It is not immediately obvious that this change will alter the spacing from 10px to 15px

**Comment by aliktb** [comment]
could you also add in your PR description that you are wrapping the `FormGroup` in a `<fieldset>` for accessibility requirements

**Review by aliktb — APPROVED** [review]
LGTM

File: src/FormGroup/FormGroup.scss [hunk 1]
```diff
@@ -1,3 +1,13 @@
 @import "node_modules/govuk-frontend/govuk/_base";
 @import "node_modules/govuk-frontend/govuk/objects/_form-group";
 @import "node_modules/govuk-frontend/govuk/utilities/_visually-hidden";
+
+fieldset {
```
Comment (paulfgraham) [inline src/FormGroup/FormGroup.scss:5]: You know you've done this is a global change, right? This applies to all `<fieldset />` elements, regardless of whether they're inside a `<FormGroup />` or not.

File: src/FormGroup/FormGroup.scss [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+    margin: 0;
+    border: 0;
+    padding: 0;
+}
+
+legend {
```
Comment (paulfgraham) [inline src/FormGroup/FormGroup.scss:11]: The same here - this is a global change that applies to all `<legend />` elements, regardless of whether they're inside a `<FormGroup />` or inside a `<fieldset />` element.

File: src/FormGroup/FormGroup.jsx [hunk 3]
```diff
@@ -24,10 +24,14 @@ const FormGroup = ({
   const classes = classBuilder(classBlock, classModifiers, className);
   return (
     <div {...attrs} key={id} className={classes()}>
-      <Label id={id} required={required} hideOptionalSuffix={hideOptionalSuffix}>{label}</Label>
-      {hint && <Hint id={`${id}-hint`}>{hint}</Hint>}
-      {error && <ErrorMessage id={`${id}-error`}>{error}</ErrorMessage>}
-      {children}
+      <fieldset>
+        <legend className={`govuk-fieldset__legend govuk-fieldset__legend--l`}>
```
Comment (paulfgraham) [inline src/FormGroup/FormGroup.jsx:28]: This also forces the labels to be 2.25rem, which is a huge 36px! Large labels like this should _only_ apply in the specific scenario of displaying one question per page, which means it should be configurable - i.e., something passed in and, when on a form, driven by the individual component JSON in the form being displayed.

**Pull request merged**